certbot/certbot/_internal/client.py

```diff
@@ -612,24 +613,19 @@ def apply_enhancement(self, domains, enhancement, options=None):
 
 
         """
-        msg = f"Could not set up {enhancement} enhancement"
+        enh_label = options if enhancement == "ensure-http-header" else enhancement
+        msg = f"Reverted the configuration after failing to apply the {enh_label} enhancement."
```

Comment:
In my opinion, this is a status message that doesn't need to be shown. Reverting the enhancement is internal and the expected outcome in this case. Why does the user need to know?

My vote is to just not print anything in this call to `self._recovery_routine_with_msg`, but I'd like to get a second opinion.

Reply:
It looks like you added this message? Am I missing something here? Is the question, "Can we leave this message empty, since I removed the redundant message?" If so, yes.

Reply:
> It looks like you added this message? Am I missing something here? Is the question, "Can we leave this message empty, since I removed the redundant message?" If so, yes.

Sorry, I shouldn't have done two things at once, that's confusing.

I changed the status message from `f"Could not set up {enhancement} enhancement"`, which was redundant with all the  error logging, to the current message, which at least has a distinct meaning.

Further to that, my question is whether we can just not have any message at all, with the reasoning that it doesn't qualify under our criteria for what status messages should be shown. 

Edit: Moved this comment after I posted it in the wrong part of the review, sorry.

Reply:
Yeah, I agree, this message does not need to be shown.